protocols/README.md

```diff
@@ -0,0 +1,7 @@
+# Documentation/Protocols
+
+## Prerequisites
+
+1. [OPTIONAL/RECOMMENDED] Use a true linux/unix install. *GNUwin32* and similar software collections may not behave correctly.
+
+2. Make sure that `groff` is correctly installed. (`apt install groff` or similar) *As per issue #11*
```

Comment:
I would add a bit of explanation on Ubuntu half installs of groff.

Reply:
@JetseVerschuren I'll do that after the merge (should happen today or tomorrow)